angular/core/components/thread_page/thread_page.haml

```diff
@@ -2,81 +2,44 @@
-        .lmo-placeholder.lmo-hint-text.table-cell{translate: "start_proposal_card.helptext"}
-        .start-proposal-card__start-proposal-button.lmo-no-print
-          %start_proposal_button.table-cell.thread-start-proposal-button{discussion: "threadPage.discussion"}
-      %current_proposal_card{in-view: "threadPage.proposalInView($inview)", in-view-options: "{debounce: 200}", ng-if: "threadPage.discussion.hasActiveProposal()", proposal: "threadPage.discussion.activeProposal()"}
-      %previous_proposals_card{discussion: "threadPage.discussion"}
-      %members_card{group: "threadPage.discussion.group()"}
-
-    .lmo-thread-column-left
-      %activity_card{discussion: "threadPage.discussion", active-comment-id: "threadPage.requestedCommentId"}
-      %comment_form.lmo-no-print{discussion: "threadPage.discussion"}
-      %outlet{name: "thread-page-column-right"}
+    .thread-page__main-content--small{ng-if: "threadPage.windowIsSmall"}
+      .thread-page__context
+        %context_panel{discussion: "threadPage.discussion"}
+      .thread-page__proposals
+        %section.start-proposal-card.lmo-no-print{in-view: "threadPage.proposalButtonInView($inview)", in-view-options: "{debounce: 200}", ng-if: "threadPage.canStartProposal()", aria-label: "{{ 'start_proposal_card.title' | translate }}"}
+          %h2.lmo-card-heading{translate: "common.models.proposal"}
+          .lmo-placeholder.lmo-hint-text.table-cell{translate: "start_proposal_card.helptext"}
+          .start-proposal-card__start-proposal-button.lmo-no-print
+            %start_proposal_button.table-cell.thread-start-proposal-button{discussion: "threadPage.discussion"}
+        %current_proposal_card{in-view: "threadPage.proposalInView($inview)", in-view-options: "{debounce: 200}", ng-if: "threadPage.discussion.hasActiveProposal()", proposal: "threadPage.discussion.activeProposal()"}
+        %previous_proposals_card{discussion: "threadPage.discussion"}
+      .thread-page__activity
+        %activity_card{discussion: "threadPage.discussion", active-comment-id: "threadPage.requestedCommentId"}
+        %comment_form.lmo-no-print{discussion: "threadPage.discussion"}
+        %outlet{name: "thread-page-column-right"}
+      .thread-page__outlet
+        %outlet{name: "before-thread-page-column-right", model: "threadPage.discussion"}
+      .thread-page__members
+        %members_card{group: "threadPage.discussion.group()"}
+
+    .thread-page__main-content--large{ng-if: "threadPage.windowIsLarge"}
+      .lmo-thread-column-left
+        %context_panel{discussion: "threadPage.discussion"}
+      .lmo-thread-column-right
+        %outlet{name: "before-thread-page-column-right", model: "threadPage.discussion"}
+        %section.start-proposal-card.lmo-no-print{in-view: "threadPage.proposalButtonInView($inview)", in-view-options: "{debounce: 200}", ng-if: "threadPage.canStartProposal()", aria-label: "{{ 'start_proposal_card.title' | translate }}"}
```

Comment:
Seems like we should pull this out into a directive as well, to avoid duplicating it.